akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -1610,9 +1628,9 @@ abstract class TimerGraphStageLogic(_shape: Shape) extends GraphStageLogic(_shap
   final protected def scheduleOnce(timerKey: Any, delay: FiniteDuration): Unit = {
     cancelTimer(timerKey)
     val id = timerIdGen.next()
-    val task = interpreter.materializer.scheduleOnce(delay, new Runnable {
-      def run() = getTimerAsyncCallback.invoke(Scheduled(timerKey, id, repeating = false))
-    })
+    val callback = getTimerAsyncCallback
```

Comment:
Do we know if `getTimerAsyncCallback` still needs that pseudo-lazy structure? Could it be just a `val` instead?

Reply:
Not sure, saves allocating it if the timer is never used in a stage that sometimes uses a timer perhaps?